### PR TITLE
backport: chia 370, 371

### PR DESCRIPTION
**Pull request by UdjinM6**
replaces https://github.com/dashpay/bls-signatures/pull/67

### CODE REVIEW

File: include/dashbls/schemes.hpp [hunk 1]
```diff
@@ -186,7 +187,7 @@ class AugSchemeMPL : public CoreMPL {
                          const G2Element& signature) override;
 };
 
-class PopSchemeMPL : public CoreMPL {
+class PopSchemeMPL final : public CoreMPL {
```
Comment (knst) [inline include/dashbls/schemes.hpp:190]: missing `class LegacySchemeMPL final : public CoreMPL {`
that is not presented in Chia's library.

**Review by knst — CHANGES_REQUESTED** [review]

File: include/dashbls/schemes.hpp (hunk 1 above)
Comment (UdjinM6) [reply include/dashbls/schemes.hpp:190]: Thanks! added in 72f0ad2

**Review by knst — APPROVED** [review]
utACK

**Comment by UdjinM6** [comment]
@PastaPastaPasta ping

**Pull request merged**